src/components/recipeComponent.js

```diff
@@ -1,10 +1,27 @@
 // Create image component
 
+// Errors
+// const recipeTitleError = "No title in recipeComponent";
+// const recipeTextError = "No text in recipeComponent";
+// const recipeImageError = `No ${variable} in recipeComponent`;
+const recipeErrors = `No ${variable} in recipeComponent`;
+
 // Config file
 import { pathImages } from "../config.js";
 
+// Checking for errors start
+function checkingForErrors(variable) {
```

Comment:
if we named this method as `checking` then we need to do checks inside.

you can do if(!variable) inside of it and it will remove few lines of code